nuget/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.Package.csproj

```diff
@@ -46,6 +46,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\Microsoft.CodeAnalysis.VersionCheckAnalyzer\Microsoft.CodeAnalysis.VersionCheckAnalyzer.Package.csproj" ReferenceOutputAssembly="false" />
```

Comment:
❔ Why does this need to be added as part of this pull request?

Reply:
It was just a bug where the FxCop analyzers package already references the VersionCheckAnalyzer package, but there is no P2P reference. I just noticed it while refactoring.